kokoro/linux/build.sh

```diff
@@ -81,6 +81,46 @@ do
 done
 echo $(date): Smoketests completed.
 
+set -e
+
 # Build the release packages.
 mkdir $BUILD_ROOT/out
 $SRC/kokoro/linux/package.sh $BUILD_ROOT/out
+
+##
+## Test capture and replay of the Vulkan Sample App.
+##
+
+# Install the Vulkan loader and xvfb (X virtual framebuffer).
+sudo apt-get -qy install libvulkan1 xvfb
+
+# Get prebuilt SwiftShader.
+curl -fsSL -o swiftshader.zip https://github.com/google/gfbuild-swiftshader/releases/download/github%2Fgoogle%2Fgfbuild-swiftshader%2F0bbf7ba9f909092f0328b1d519d5f7db1773be57/gfbuild-swiftshader-0bbf7ba9f909092f0328b1d519d5f7db1773be57-Linux_x64_Debug.zip
+unzip -d swiftshader swiftshader.zip
+
+# Use SwiftShader.
+export VK_ICD_FILENAMES="$(pwd)/swiftshader/lib/vk_swiftshader_icd.json"
+export VK_LOADER_DEBUG=all
+
+# Just try running the app first.
+
+# Allow non-zero exit status.
+set +e
+
+xvfb-run -e xvfb.log -a timeout --preserve-status -s INT -k 10 5 bazel-bin/cmd/vulkan_sample/vulkan_sample
```

Comment:
`-k 10` should probably be `-k 1`, it is **extra** time (in seconds) before sending sigkill, i.e.:
`timeout -s INT -k 1 5 foo` runs `foo` for 5s, sends SIGINT, waits an **extra** 1s for `foo` to terminate, and sends SIGKILL otherwise.

From the man page:
```
       -k, --kill-after=DURATION
              also send a KILL signal if COMMAND is still running
              this long after the initial signal was sent
```

Reply:
Thanks! I changed to 5, which is what I intended.